tests/DependencyInjection/AbstractDoctrineExtensionTestCase.php

```diff
@@ -629,10 +631,13 @@ public function testCacheConfig(string|null $expectedClass, string $entityManage
         if ($expectedClass === null) {
             $this->assertNull($cache);
         } else {
+            /** @var class-string $expectedClass */
```

Comment:
This looks useless to me. phpstan should be able to infer it thanks to the `if/else`